messenger_users/views.py

```diff
@@ -79,6 +80,35 @@ def confirm_linkedin_profile(request):
 # def browse_saved_jobs(request):
 #     pass
 
+def search_linkedIn_profiles(request):
+    data = json.loads(request.body)
+    messenger_id = data['messenger_id']
+    username = data['linkedin_username']
+
+    searchObject = Search()
+    
+    urlArray = searchObject.get_user_profile_suggestions(username)
+    
+    gallery_message = GalleryMessage("square")
+    
+    for addr in urlArray:
+        image_url = 'https://blog.herzing.ca/hubfs/becoming%20a%20programmer%20analyst%20lead-1.jpg'
+        apply_button = UrlButton("It's me!", addr)
+        gallery_card = GalleryCard("Sample name", "", image_url,
+                buttons=[apply_button])
+
+        gallery_message.add_card(gallery_card)
+
+    next_page = QuickReply("Next Page", block_name="linkedInSearch")
+    next_page.set_attribute("result_page", str(offset+1))
+    gallery_message.add_quick_reply(next_page)
```

Comment:
Don't need next page quickreply... that's a specific thing for the other API that it's from. offset is not a defined a variable in this case, and the code will break. careful about copying code